src/services/tracker_service.js

```diff
@@ -0,0 +1,20 @@
+import baseService from './base_service';
+
+export default (easypostClient) =>
+  class TrackerService extends baseService(easypostClient) {
+    static _name = 'Tracker';
+
+    static _url = 'trackers';
+
+    static key = 'tracker';
+
+    /**
+     * Create trackers in bulk.
+     * @param {object} params
+     */
+    static async createList(params = {}) {
+      const newParams = { trackers: params };
+      const url = 'trackers/create_list';
+      await easypostClient.post(url, newParams);
```

Comment:
```suggestion
    static async createList(params = {}) {
      const wrappedParams = { trackers: params };
      const url = 'trackers/create_list';
      const response = await easypostClient.post(url, wrappedParams);
      
      return response;
```
This endpoint is not documentated, I know you didn't write this but it might need a small tweak.

Reply:
It doesn't return anything so like the `delete` functions, you'd just call it and can't do anything with the response since it's empty.

Reply:
Ah, yes. I forgot this endpoint doesn't return anything